oneflow/user/kernels/nccl_logical_fusion_kernel.cpp

```diff
@@ -655,14 +656,15 @@ void NcclLogicalFusionKernel::Compute(user_op::KernelComputeContext* ctx,
   ccl::CclComm ccl_comm = kernel_state->ccl_comm();
 
   // do nccl compute in group
-  OF_NCCL_CHECK(ncclGroupStart());
+  // TODO:(zhaoluyang) replacre ncclGroupStart/ncclGroupEnd with ccl CclGroupStart/CclGroupEnd
+  // OF_NCCL_CHECK(ncclGroupStart());
```

Comment:
`ncclGroupStart` `ncclGroupEnd` wraps the code segment and has three functions: 
- 1. Sequential execution (guaranteeing that each communication operation is submitted to the stream in order) 
- 2. Communication operations are non-blocking 
- 3. Reduce synchronization overhead (as a whole group, avoid the synchronization required for each single communication operation)